src/style.css

```diff
@@ -1,3 +1,69 @@
 body {
+  font-family: Roboto, sans-serif;
```

Comment:
Consider adding a fallback font to ensure consistent appearance if Roboto fails to load.